testsuite/tests/parallel/pingpong.ml

```diff
@@ -1,4 +1,13 @@
-(* TEST *)
+(* TEST
+ no-tsan; (* TSan detects the intentional data race *)
+ flags = "-g";
```

Comment:
Why did you have to add this flag?

Reply:
This must be a left over from debugging. Fixed in 4fe058fd1c, thanks!